PR for Issue#9

daniel-swinehart opened the pull request:
Portfolio with TypeScript compiled, built, and deployed to Netlify [here](https://daniel-issue9.netlify.app/).

Close Issue #9

daniel-swinehart committed:
install parcel

daniel-swinehart committed:
ts folder setup

daniel-swinehart committed:
basic ts logic complete

daniel-swinehart committed:
added comments to ts file

daniel-swinehart committed:
ts config files

daniel-swinehart committed:
trying to fix dist folder issue

daniel-swinehart committed:
parcel data

daniel-swinehart committed:
file structure change

daniel-swinehart committed:
moved all to src folder, updated package.json

daniel-swinehart committed:
restructure folder

daniel-swinehart committed:
update package build

daniel-swinehart committed:
updates

daniel-swinehart committed:
I think i have folder structure rignt now for netlify

daniel-swinehart committed:
updated gitignore with parcel-cache

daniel-swinehart committed:
trying for parcel

axecopfire commented on .gitignore:
```diff
@@ -74,6 +74,7 @@ typings/
 
 # parcel-bundler cache (https://parceljs.org/)
 .cache
+.parcel-cache
```
- Can you remove `.parcel-cache` from this `.gitignore`
- Then delete the `.parcel-cache` folder. 
- Then commit that change
- Then add `.parcel-cache` back to the `.gitignore`
- Then add that change

daniel-swinehart replied:
will do 👍

axecopfire commented on issue9/src/issue9.ts:
```diff
@@ -0,0 +1,37 @@
+console.log(addToArray(objArray, 5));
+
+//Created empty array with array type 'object'
+const newObjArray: {name: string, address: string, phone: string} [] = [];
```
I see that you're doing all these types inline. However, you're losing one of the benefits of typescript and that's consistency.

Declare a type at the top as `{name: string, address: string, phone: string} []` and reference it in all these other places.

daniel-swinehart replied:
I see what you mean, and that makes sense. I'll make the change

daniel-swinehart replied:
made changes.

axecopfire commented on issue9/src/projects.html:
```diff
@@ -0,0 +1,57 @@
+                <div class="project-img-container">
+                    <p>Project image will go here</p>
+                </div>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proiden ...
```
I'm not a fan that you're still using `lorem ipsum`. I believe you had a design that had content. We shouldn't be 4 weeks after that design still doing the same stuff. Ur page does look close-ish to your design. However, I'm also not a fan of the copy paste folders `issue9`. I'd rather you just had 1 website that you were consistently working on instead of pushing to different weeks. I added to the Feedback doc that there needs to be a better explanation around how the folder structure should work. Another thing about adding the folders for each week is that this looks like a 33 file change, where you're really only updating 2 files it seems. That makes it harder on the code reviewers to review.

daniel-swinehart replied:
I updated the design, added content and made a logo.

daniel-swinehart committed:
updated type in .ts file

daniel-swinehart committed:
test gitignore

daniel-swinehart committed:
dont need

jeromehardaway commented:
Hey @daniel-swinehart any movement on this?

daniel-swinehart commented:
Yes, I updated the gitignore file and updated my .ts file to declare a type and use throughout the code. I'm working on updating my portfolio html as shuster requested. This last week I was traveling , but I will be working on it tonight and the test of the week.

daniel-swinehart committed:
uopdated google map

daniel-swinehart committed:
email form/social media link css

daniel-swinehart committed:
css for contact page, new icon images

daniel-swinehart committed:
added imgs css for projects page

daniel-swinehart committed:
updated project page content

daniel-swinehart committed:
updated profile pic and slideshow

daniel-swinehart committed:
slideshow css

daniel-swinehart committed:
updated logo

daniel-swinehart committed:
twitter feed added

daniel-swinehart committed:
css for twitter feed

daniel-swinehart committed:
scaled and compressed slideshow img

jeromehardaway commented:
Still needs a little work on mobile.![image](https://github.com/Vets-Who-Code/js-web-path-daniel-swinehart/assets/7096246/6aa20daa-2628-4258-a998-509797687989)

daniel-swinehart committed:
fixed foot for lat overflow issue

daniel-swinehart committed:
working hamburger menu

daniel-swinehart committed:
added mobile nav

daniel-swinehart committed:
Merge branch 'main' into issue#9
